extra newline causes migration to fail in MacOS

Due to the echo statement in `url_escape` migration is failing in MacOS.

https://github.com/G-Research/yunikorn-history-server/blob/7144c3127f10698a32465da0b4b86e65521e05d2/Makefile#L132-L134

**Sample Error**
```
/Users/sudiptobaral/github.com/sudiptob2/yunikorn/yunikorn-history-server/bin/tooling/migrate -path migrations -database        postgres://-n+postgres%0A:-n+psw%0A@-n+127.0.0.1%0A:30002/-n+yhs%0A?sslmode=disable up
error: parse "postgres://-n+postgres%0A:-n+psw%0A@-n+127.0.0.1%0A:30002/-n+yhs%0A?sslmode=disable": invalid URL escape "%0A"
make[1]: *** [Makefile:149: migrate] Error 1
make[1]: Leaving directory '/Users/sudiptobaral/github.com/sudiptob2/yunikorn/yunikorn-history-server'
Error while applying migrations. Retrying in 10 seconds...
```

--- a/Makefile
+++ b/Makefile
@@ -130,7 +130,7 @@ define yq_get_db
 endef
 
 define url_escape
-    $(shell echo -n $(1) | go run hack/url/main.go)
+    $(shell printf "%s" $(1) | go run hack/url/main.go)
 endef
 
 

